docs/docs/examples/guides/05-jtc-guide.md

```diff
@@ -0,0 +1,589 @@
+---
+sidebar_position: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import jtcGuideHW from './assets/jtc-guide-hw.png'
+import jtcGuideParameters from './assets/jtc-guide-parameters.png'
+import jtcGuideRecordFrame from './assets/jtc-guide-record-frame.gif'
+import jtcGuideCreateFrame from './assets/jtc-guide-create-frame.gif'
+import jtcGuideRecordJoint from './assets/jtc-guide-record-joint.gif'
+
+# A guide on Joint Trajectory workflows
+
+In our concepts page for the [Joint Trajectory Controller](../../concepts/05-building-blocks/04-controllers/jtc.md)
+(JTC) we already covered why trajectory controllers are quite often needed in robotics. In short, executing trajectories,
+in joint- or Cartesian-space, is quite often a fundamental piece of a robotics application workflow. Whether you need to
+address pick and place tasks, or to simply move through predetermined locations, you are in need of a controller that is
+able to traverse space accurately and timely.
+
+While in ["A joint trajectory execution example"](./04-jtc-example.md) we covered a basic example of how JTC can be
+used, here we will focus more on its integration with AICA Studio and suggest workflows and/or parametrizations that
+might aid you when building big applications.
+
+## Setting up your environment
+
+JTC has a plethora of parameters that can be set to alter its performance according to an application's requirements.
+Let us start by creating a new application that we can use as a reference point for this guide.
+
+First, start AICA Launcher and create a configuration with at least `AICA Core v4.4.1`. For the remainder of this guide,
```

Comment:
Is that really a code block, don't we want to say "a configuration with AICA Core version 4.4.1 or higher"?